.teamcity/src/main/kotlin/configurations/FunctionalTest.kt

```diff
@@ -38,7 +38,7 @@ class FunctionalTest(
     }
 
     applyTestDefaults(
-        model, this, testTasks, notQuick = !testCoverage.isQuick, os = testCoverage.os,
+        model, this, testTasks, dependsOnQuickFeedbackLinux = !testCoverage.isQuick, os = testCoverage.os,
```

Comment:
So the downstream tests continue to depend on QuickFeedbackLinux. And that will fail when sanity check fails, right? I thought the idea was that the downstream test would only depend on compileAll and not on anything else from `QuickFeedbackLinux`. Am I missing something here?

Reply:
Now this is done. All tests in PRF stage depends on CompileAll only, except Gradleception.